charts/replicated-library/values.yaml

```diff
@@ -33,3 +33,200 @@ defaults:
       successThreshold: 1
       failureThreshold: 5
     startupProbe: {}
+
+troubleshoot:
+  replicated:
+    type: SupportBundle
+    enabled: false
+    uri: https://raw.githubusercontent.com/replicatedhq/troubleshoot-specs/main/in-cluster/default.yaml
+    collectors:
+      - clusterInfo: {}
+      - clusterResources: {}
+      - ceph: {}
+      - longhorn: {}
+      - exec: # this is removable when we don't need to support kots <= 1.87
+          args:
+            - "-U"
+            - kotsadm
```

Comment:
I like it and in-fact if the new KOTS is actually a helm chart it can install it's own bundles. We might not even need a kots spec at all.

Both versions of KOTS should install their own support bundles, we really should only need to add more generic stuff that we want to be sure is there. ClusterInfo, and such.